apps/site/lib/site_web/templates/customer_support/index.html.eex

```diff
@@ -1,125 +1,84 @@
-              MBTA<br />
-              10 Park Plaza<br />
-              Suite 5610<br />
-              Boston, MA 02116
-            </p>
-          </div>
-        </div>
-        <div class="row">
-          <div class="col-xs-12 col-sm-6">
-            <h2>Request Public Records</h2>
-            <p>Members of the public have the right to request copies of MBTA documents, with certain exceptions. Fees apply and may vary depending on the request.</p>
-            <p>
-              <%= link to: "https://massachusettsdot.mycusthelp.com/WEBAPP/_rs/supporthome.aspx?lp=3&COID=64D93B66", target: "_blank", class: "c-call-to-action" do %>
-                Submit a public records request to the MBTA
-              <% end %>
-            </p>
-            <p>You can also email <a class="c-call-to-action" href="mailto:mbta.rao@mbta.com">mbta.rao@mbta.com</a> with your request.</p>
-          </div>
-          <div class="col-xs-12 col-sm-6">
-            <h2>Report Fraud, Waste, or Abuse</h2>
-            <p>The Internal Special Audit Unit within the Office of the Inspector General monitors the quality, efficiency, and integrity of MassDOT programs.</p>
-            <p>
-              <%= link to: "https://www.mass.gov/service-details/massdot-fraud-waste-and-abuse", target: "_blank", class: "c-call-to-action" do %>
-                Report suspected fraud, waste, or abuse
-              <% end %>
-            </p>
-          </div>
-        </div>
-        <div class="row">
-          <div class="col-xs-12 col-sm-6">
-            <h2>Join Us</h2>
-            <p>Check out the exciting roles we're hiring for and find out how to start your career at the MBTA.</p>
-            <p>
-              <%= link to: "/careers", class: "c-call-to-action" do %>
-                Learn about MBTA career opportunities
-              <% end %>
-            </p>
-            <p>
-              <%= link to: "https://jobs.lever.co/mbta", target: "_blank", class: "c-call-to-action" do %>
-                View Customer Technology opportunities
-              <% end %>
-            </p>
+        <div class="col-xs-12 support-form-error <%= if !show_error_message(@conn), do: "hidden-xs-up" %>" tabindex="-1">
+          <div class="support-error" id="<%= if show_error_message(@conn), do: "support-result" %>">
+            <%= fa "exclamation-circle" %>
+            We had an issue submitting your feedback -- please call us at the Main Hotline below, or try again in a few minutes.
           </div>
         </div>
       </div>
+      <%= if @show_form do %>
+        <%= render "_form.html", forward_assigns(@conn) %>
+      <% end %>
+
+    </div>
+
+    <div class="col-xs-12 col-md-4">
+
+      <%= SiteWeb.PartialView.ExpandableBlock.render_expandable_block(
+          Map.merge(assigns, %{
+            header: %{text: "Call Us", iconSvgText: nil},
+            id: "call_us",
+            show_expanded: true,
+            view_template: "_call_us.html"
+      })) %>
+
```

Comment:
Part of me wonders if these repetitious calls can be handled a little more elegantly on the Elixir side (with pattern matching perhaps?!), but this way is also perfectly great.

Reply:
I thought so as well, too repetitive - I'll see if I can optimize this.